wallet/storage/src/internal/store_tx.rs

```diff
@@ -52,13 +55,50 @@ mod well_known {
     }
 
     declare_entry!(StoreVersion: u32);
+    declare_entry!(EncryptionKeyKdfChallenge: KdfChallenge);
+}
+
+#[derive(PartialEq, Clone)]
+pub enum EncryptionState {
+    Locked,
+    Unlocked(Option<SymmetricKey>),
```

Comment:
What does `Unlocked<None>` mean?

If it means Unlocked without encryption, can we make a third state, `Unencrypted` to cover that?

Reply:
You know what, I thought about it. Keep it like this. Just please add some documentation explaining what `Unlocked(None)` means.

Reply:
added comments